### PR TITLE
Scaling back up the 4.6 testing

### PR DESCRIPTION
**Pull request by bradmwilliams**
Reverting 4.6 specific changes from #7807 to bring 4.6 testing back up to appropriate levels.

### CODE REVIEW

File: core-services/release-controller/_releases/release-ocp-4.6.json [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-  "expires":"12h",
-  "maxUnreadyReleases": 1,
-  "minCreationIntervalSeconds": 7200,
+  "expires":"168h",
+  "maxUnreadyReleases": 2,
+  "minCreationIntervalSeconds": 2400,
   "referenceMode": "source",
   "pullSecretName": "source",
+  "check":{
+￼   "OCP and Origin images should match": {
```
Comment (wking) [inline core-services/release-controller/_releases/release-ocp-4.6.json:12]: you have some kind of weird prefix char at the start of this line.

Comment (bradmwilliams) [reply core-services/release-controller/_releases/release-ocp-4.6.json]: Yeah, I have no idea how that got there!  It's gone now!

**Comment by wking** [comment]
.[CI][1] is suggesting you run `make release-controllers`, although it is frustratingly not sharing the changes it would like to see, just:

```
 ERROR: Run the following command to re-generate the release controller configurations, run:
ERROR: $ make release-controllers
ERROR: The following differences were found:
9a10
> 247c3058d26b4d6dd723bc3a459e0fc1  .//core-services/release-controller/_releases/priv/release-ocp-4.6-ppc64le.json
10a12
> 259f7a894ce167587a91c37c0b7adcbb  .//core-services/release-controller/_releases/priv/release-ocp-4.6-s390x.json
...
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/10499/pull-ci-openshift-release-master-release-controller-config/1288180663668707328#1:build-log.txt%3A139

**Comment by wking** [comment]
Ah, looks like you addressed that with ec8eca1865 -> 1e1c7aa077.

**Comment by bradmwilliams** [comment]
> This is not restoring `gcp-serial` or `upgrade` to `release-ocp-4.6-ci.json` (#7807 removed them as well), but I'm fine with that. There are a few more divergences from a clean revert in:
> 
> ```
> $ diff -u <(git diff --color=always origin/pr/7807..origin/pr/7807^) <(git show --color=always origin/pr/10499)
> ```
> 
> But all of the divergence makes sense to me.
> 
> /lgtm

Yeah, I was fearful about adding them since the latest code doesnt even have the **gcp** job that remained post #7807.  I can certainly add them back, but didnt want to risk breaking something that had been removed on purpose.

**Pull request merged**